electrumx/lib/coins.py

```diff
@@ -3321,3 +3321,22 @@ class BitcoinRoyale(Coin):
     ESTIMATE_FEE = 0.00001
     RELAY_FEE = 0.00001
     DAEMON = daemon.FakeEstimateFeeDaemon
+
+
+class BitcoinVaultRegTest(Coin):
+    NAME = "BitcoinVaultRegTest"
+    SHORTNAME = "BTVRT"
```

Comment:
`BTCVRT` should be used instead of `BTVRT`

Reply:
done :)